docs/src/content/docs/basics/terminology.mdx

```diff
@@ -24,20 +28,12 @@ An assertion made about a *subject* in a *context*.
 
 ## Connection
 
-A data exchange relationship between (a) an *identity agent* and (b) an *app* or *agent*. A *connection*'s state is stored by one or more *contexts*. 
+A data exchange relationship between (a) an *identity agent* and (b) an *app*. A *connection*'s state is stored by one or more *contexts*.
 
 ## Connector
 
 A software component that implements one of potentially several aspects of a *connection*. A *connector* implements an interface (abstraction) exposed by the core of an *identity agent*. It is associated with a single *context*. It may receive data from an *app*, may store receieved data in a *context*, may read data from a *context* and may transmit data to an *app*.
 
-## Agent SDK
-
-A software development kit that (i) provides the core logic of an *identity agen*t and (ii) enables a *local* *app* to become interoperable with the *Mee data network*. The *agent SDK* manages *personal information* that may be (i) created from the user's data entry into their *identity agent* (ii) created by an app (or co-created by an *app* during interactions with the user) or (iii) received over the *Mee data network* from another *identity* *agent*.
-
-## Agent Service
```

Comment:
Just moved this section upper alphabetically

Reply:
I agree with this change